app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,33 @@
+<section class="section_frm">
+<div class="div_title">
+<h1>Welcome to NomadDev!</h1>
+</div>
+
+<div class="div_log_frm">
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Your email", class: "input"%><br>
+  </div>
+
+  <div class="field">
+    <%= f.password_field :password, autocomplete: "current-password", placeholder: "Your password", class: "input"%><br>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %><br>
+    </div>
+  <% end %>
+
+  <div class="actions div_buttons">
+    <%= f.submit "Log in", class: "button login_btn"%><br>
+  </div>
+ <% end %>
+</div>
+
+<div class="div_links">
+  <%= render "devise/shared/links" %>
+</div>
+</section>
```

Comment:
- [ ] Please modified the login form to match the [wireframes](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md) and their styling.
![image](https://user-images.githubusercontent.com/55596970/200728152-52cde040-b90a-49fb-9b3f-38899a2ce234.png)